parlai/core/teachers.py

```diff
@@ -408,7 +408,7 @@ def act(self):
             action = action.copy()
             labels = action.pop('labels')
             if not self.opt.get('hide_labels', False):
-                action['eval_labels'] = labels
+                action.force_set('eval_labels', labels)
```

Comment:
hmm, so this would happen if the action already has eval_labels? 

this is confusing to me because the above `if` statement checks if there are `labels` in the action. in which case would we have both `labels` and `eval_labels`?

Reply:
oh yeah I didn't look at the context at all. This is only triggered by the temp fromfile task I was using, shouldn't be affecting people usually. I'll get rid of it.